### PR TITLE
Fixing issues bug bash 17 06

### PR DESCRIPTION
**Pull request by Pbarbosa92**
This PR addresses the issues detected during the 17-06 bug bash.

Most of the issues were related with outputs. Ex: inductiva resources terminate --all

### CODE REVIEW

File: inductiva/logs/log.py [hunk 1]
```diff
@@ -42,6 +44,12 @@ def handle_uncaught_exception(exc_type, exc_value, exc_traceback):
     if issubclass(exc_type, KeyboardInterrupt):
         sys.__excepthook__(exc_type, exc_value, exc_traceback)
         return
+    if issubclass(exc_type,
+                  exceptions.ApiException) and (exc_value.status >= 400 and
+                                                exc_value.status < 500):
```
Comment (sergiomsantos) [inline inductiva/logs/log.py:49]:
```suggestion
    if issubclass(exc_type, exceptions.ApiException) and \
        400 <= exc_value.status  < 500:
```

**Review by sergiomsantos — COMMENTED** [review]
just a minor comment

**Review by sergiomsantos — APPROVED** [review]

**Comment by Pbarbosa92** [comment]
@bgregoinductiva @Franreno @luispcunha Can any of you do some QA on this PR?
Test `inductiva resources terminate --all --yes` to check we generate less text
Test `inductiva.tasks.get()[0].get_info()` to check it will print the str instead of the object.
And test `inductiva tasks list --task-id 5dm4qty5srrd9rtje07fer2vq` to trigger the `Task ID not present in the list of tasks owned by the user.` Exception

**Comment by Franreno** [comment]
# QA

### Terminate all

Worked correctly ✅

```
➜  inductiva resources terminate --all -y
Successfully requested termination of api-pfpqn8h9c57nj13nixssb2xgs.
Successfully requested termination of api-9tvig3b4emfxxh7d0s81uem9q.
Successfully requested termination of api-o2rp27nci4f4m2nf4qtkravhd.
Successfully requested termination of api-2t2rxsy051naqqinbh36txk3w.
Successfully requested termination of api-nu3ze7gb0s4gg3nb1vwufcknm.
Successfully requested termination of api-abypeadwzqau2ugu1bui5gbgx.
Successfully requested termination of api-hiknhwx4s2esnssg2ibvzehxw.
Successfully requested termination of api-bww0gy3w7s9ry9ymnosxp3dsn.
Successfully requested termination of api-oymzpqyjh2javtxw3s6xh47sn.
Successfully requested termination of api-inwhqvymqs77tp1u0dfmgt5b0.

FREED BY RESOURCES     CURRENT USAGE     MAX ALLOWED
 cost_per_hour        2.39128                0.09448           270
 total_num_vcpus      48                     8                 1500
 total_num_machines   12                     2                 100
 ```
 
 ---

### get_info()

Worked correctly ✅

```
➜  python3                                                                  
Python 3.11.5 (main, Sep 11 2023, 08:31:25) [Clang 14.0.6 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import inductiva
>>> inductiva.tasks.get()[0].get_info()

Task status: success
Wall clock time:  0:02:59
Time breakdown:
	Input upload:              0.38 s
	Time in queue:             0:02:09
	Container image download:  3.32 s
	Input download:            0.05 s
	Input decompression:       0.04 s
	Computation:               43.47 s
	Output compression:        2.29 s
	Output upload:             0.32 s
Data:
	Size of zipped output:    13.15 MB
	Size of unzipped output:  40.13 MB
	Number of output files:   439
```

---

### task_list

Worked correctly ✅

```
➜  inductiva tasks list --task-id 5dm4qty5srrd9rtje07fer2vq
Task ID not present in the list of tasks owned by the user.
franreno in ~/inductiva/bugbash4 
➜  inductiva tasks list --task-id inductiva                
Invalid task ID format.
```

**Pull request merged**